test/e2e/e2e_suite_test.go

```diff
@@ -29,7 +29,7 @@ func handleFlags() {
 
 var viperConfig = flag.String("viper-config", "", "The name of a viper config file (https://github.com/spf13/viper#what-is-viper). All e2e command line parameters can also be configured in such a file. May contain a path and may or may not contain the file suffix. The default is to look for an optional file with `e2e` as base name. If a file is specified explicitly, it must be present.")
 
-func init() {
+func realInit() {
```

Comment:
I think it might be easier to just fold this in to `TestMain`

Reply:
ok will do